docs/data-ai/capture-data/lorawan.md

```diff
@@ -0,0 +1,522 @@
+---
+title: "Create a LoRaWAN network"
+linkTitle: "Create a LoRaWAN network"
+weight: 60
+type: "docs"
+description: "Configure a gateway and nodes to communicate over the LoRaWAN protocol."
+tags: ["sensor", "components", "lorawan", "gateway", "node"]
+icon: true
+images: ["/icons/components/sensor.svg"]
+---
+
+[LoRaWAN (Long Range Wide Area Network)](https://lora-alliance.org/) enables sensor communication spanning kilometers with minimal power usage.
+
+To collect LoRaWAN sensor data with Viam, use the [`lorawan`](https://app.viam.com/module/viam/lorawan) {{< glossary_tooltip term_id="module" text="module" >}}.
+You can find the module's source code in the [lorawan GitHub repository](https://github.com/viam-modules/lorawan).
+
+## Hardware requirements
+
+- A [supported LoRaWAN gateway](https://github.com/viam-modules/lorawan?tab=readme-ov-file#lorawan-gateway-models-provided)
+- A [supported LoRaWAN node](https://github.com/viam-modules/lorawan?tab=readme-ov-file#lorawan-sensor-models-provided)
+
+## Architecture
+
+You can use Viam to create LoRaWAN networks containing one gateway and multiple nodes.
+In a LoRaWAN network, information flows in two directions:
+
+- **uplinks** transmit from nodes to gateways
+- **downlinks** transmit from gateways to nodes
+
+![A LoRaWAN network consisting of a single gateway and multiple nodes](/components/sensor/lorawan.png)
+
+### Nodes
+
+Nodes measure data like any other sensor.
+Instead of physically connecting to a machine over GPIO pins or USB, nodes communicate with your machine using the wireless LoRaWAN protocol.
+LoRaWAN supports communication over distances of up to 10 kilometers.
+Nodes typically run off of battery power; a single coin cell battery can power a node for weeks or months.
+
+### Gateway
+
+Gateways collect data generated by LoRaWAN nodes.
+A gateway is a device that communicates with nodes over LoRaWAN.
```

Comment:
```suggestion
The gateway is the device that communicates with nodes over LoRaWAN and that connects to Viam.
```

Reply:
Incorporating this feedback in my next commit, thanks.